pkg/release/config_internal_test.go

```diff
@@ -30,6 +30,7 @@ func (s *ConfigInternalTestSuite) TestConfigHelmTypeFields() {
 		"Install",
 		"Namespace",
 		"DryRun",
+		"DryRunOption",
```

Comment:
What that option will do?

Reply:
So in 3.13.0, they have added the capability to mention dryRun to be either done at `client` side or `server` side while keeping it as a flag at the same time to keep backward compatibility.

So one use either of the three:
```
--dry-run `client`
--dry-run `server`
--dry-run
```

`--dry-run` internall in their config now becomes DryRunOption, also the earlier field DryRun is then resolved to DryRunOption eventually.

Since helmwave has a custom logic for dry running the charts hence skipped this new field as well to pass test cases